catalog/plugin/catalog-plugin-defaultsecurityattributevalues/src/test/java/org/codice/alliance/catalog/plugin/defaultsecurity/DefaultSecurityAttributeValuesPluginTest.java

```diff
@@ -13,237 +13,275 @@
-    when(mockedXss2.getValue()).thenReturn(CODEWORD_ATTRIB_VAL);
-    when(mockedAttribute1.getAttributeValues()).thenReturn(Arrays.asList(mockedXss1));
-    when(mockedAttribute2.getAttributeValues()).thenReturn(Arrays.asList(mockedXss2));
-    when(mockedAttribute1.getName()).thenReturn(USER_ATTRIB_1);
-    when(mockedAttribute2.getName()).thenReturn(USER_ATTRIB_2);
+    // then
+    verifyZeroInteractions(updateRequest);
+    assertThat(updateRequest, is(modifiedRequest));
+  }
 
-    defaultSecurityAttributeValuesPlugin =
+  @Test
+  public void testProcessDeleteRequest() {
+    // given
+    final DeleteRequest deleteRequest = mock(DeleteRequest.class);
+    final DefaultSecurityAttributeValuesPlugin defaultSecurityAttributeValuesPlugin =
         new DefaultSecurityAttributeValuesPlugin(
             new SecurityAttributes(),
-            ImmutableMap.of(CLASSIFICATION_KEY, USER_ATTRIB_1, CODEWORDS_KEY, USER_ATTRIB_2),
-            () -> mockedSubject);
-
-    when(markedMetacard.getAttribute(eq(Metacard.SECURITY))).thenReturn(securityAttributeList);
-    when(unmarkedMetacard.getAttribute(eq(Metacard.SECURITY))).thenReturn(securityAttributeList);
-    when(securityAttributeList.getValue()).thenReturn((Serializable) Collections.EMPTY_MAP);
-    when(unmarkedMetacard.getMetacardType()).thenReturn(metacardType);
-    when(metacardType.getAttributeDescriptor(any(String.class))).thenReturn(attributeDescriptor);
-
-    when(createRequest.getMetacards()).thenReturn(Collections.singletonList(unmarkedMetacard));
-    when(updateRequest.getUpdates())
-        .thenReturn(Collections.singletonList(Maps.immutableEntry("key", unmarkedMetacard)));
-
-    when(unmarkedMetacard.getMetacardType()).thenReturn(metacardType);
-    when(markedMetacard.getMetacardType()).thenReturn(metacardType);
-    when(emptySecurityAttributeList.getValue()).thenReturn(mockedEmptyAttributeMap);
-    when(mockedEmptyAttributeMap.isEmpty()).thenReturn(true);
+            createTestSystemHighAttributes(),
+            createTestInitialConfiguration());
+
+    // when
+    final DeleteRequest modifiedRequest =
+        defaultSecurityAttributeValuesPlugin.process(deleteRequest);
+
+    // then
+    verifyZeroInteractions(deleteRequest);
+    assertThat(deleteRequest, is(modifiedRequest));
   }
 
   @Test
-  public void testProcessUnmarkedMetacard() throws Exception {
-    Metacard modifiedMetacard = defaultSecurityAttributeValuesPlugin.addDefaults(unmarkedMetacard);
-    assertThat(unmarkedMetacard, not(modifiedMetacard));
+  public void testComponentManagedUpdateStrategy() {
+    // given
+    final DefaultSecurityAttributeValuesPlugin defaultSecurityAttributeValuesPlugin =
+        new DefaultSecurityAttributeValuesPlugin(
+            new SecurityAttributes(),
+            createTestSystemHighAttributes(),
+            createTestInitialConfiguration());
+
+    // when
+    final Map<String, String> newConfiguration = new HashMap<>();
```

Comment:
✏️  Nit: The map initialization should probably be in the `given` section and the `process` call in the `when` section :-)